jplag.endToEndTesting/src/main/java/de/jplag/end_to_end_testing/model/ExpectedResult.java

```diff
@@ -0,0 +1,74 @@
+package de.jplag.end_to_end_testing.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import de.jplag.JPlagComparison;
+
+/**
+ * The ExpectedResult contains all information and comparison values that are important for a test and that could be
+ * parsed from the json file.
+ */
+public class ExpectedResult {
```

Comment:
This may be a record

Reply:
Jackson JSON should be able to use records.

Reply:
class was replaced by record